rust-module/mfrc522_inner/spi.rs

```diff
@@ -1,7 +1,9 @@
 use kernel::spi::{Spi, SpiDevice};
 use kernel::{pr_info, Error, KernelResult};
 
-use super::{Mfrc522CommandByte, Mfrc522Command, Mfrc522PowerDown, Mfrc522Receiver};
+use super::{Mfrc522Command, Mfrc522CommandByte, Mfrc522PowerDown, Mfrc522Receiver};
+
+const FIFO_LEVEL_REG_FLUSH_SHIFT: u8 = 7;
```

Comment:
I think what @Skallwar meant is to put it INSIDE the implementation, like...

Reply:
At the end o we did not do that for the other one, so here is just fine for me